backend/.env.docker

```
@@ -1,10 +1,10 @@
+MONGODB_URI="mongodb://mongo-service:27017/wanderlust"
 REDIS_URL="redis://redis-service:6379"
 PORT=8080
-FRONTEND_URL="http://34.213.249.66:5173"
+FRONTEND_URL="http://34.213.249.66:31000"
 ACCESS_COOKIE_MAXAGE=120000
 ACCESS_TOKEN_EXPIRES_IN='120s'
 REFRESH_COOKIE_MAXAGE=120000
 REFRESH_TOKEN_EXPIRES_IN='120s'
 JWT_SECRET=70dd8b38486eee723ce2505f6db06f1ee503fde5eb06fc04687191a0ed665f3f98776902d2c89f6b993b1c579a87fedaf584c693a106f7cbf16e8b4e67e9d6df
-NODE_ENV=Development
+NODE_ENV=Development
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)